src/util/accountUtils.ts

```diff
@@ -267,67 +224,47 @@ const linkNamespaceToAddress = (isMultisig :boolean,cosigners: [], multisigAccou
       let privateKey = WalletUtils.decryptPrivateKey(new Password(walletPassword), accountDetails.encrypted, accountDetails.iv);
       cosignerAcc.push(Account.createFromPrivateKey(privateKey, networkType));
     });
-    
-    let txBuilder = AppState.buildTxn;
-    const aggreateBondedTx = txBuilder.aggregateBonded(innerTx);
+    const aggreateBondedTx = TransactionUtils.aggregateBondedTx(innerTx)
     const firstSigner = cosignerAcc[0]
     cosignerAcc.splice(0,1)
     const signedAggregateBondedTransaction = firstSigner.signTransactionWithCosignatories(aggreateBondedTx,cosignerAcc,Hash);
     signedTransaction = signedAggregateBondedTransaction
-    const nativeTokenNamespace = networkState.currentNetworkProfile.network.currency.namespace;
-
-    const lockingAtomicFee = networkState.currentNetworkProfileConfig.lockedFundsPerAggregate ?? 0;
-
-    const lockFundsTransaction = txBuilder.hashLock(  
-      new Mosaic(new NamespaceId(nativeTokenNamespace), UInt64.fromUint(lockingAtomicFee)),
-      UInt64.fromUint(1000),
-      signedAggregateBondedTransaction,
-    )
-    const lockFundsTransactionSigned = firstSigner.sign(lockFundsTransaction , Hash);
-    signedTransaction = signedAggregateBondedTransaction
-    multiSigAnnouce(signedAggregateBondedTransaction,lockFundsTransactionSigned)
+    const lockFundsTransaction = TransactionUtils.lockFundTx(signedAggregateBondedTransaction)
+    const lockFundsTransactionSigned = firstSigner.sign(lockFundsTransaction, Hash);
```

Comment:
please use proper name for generation hash, genHash in short. 
btw, the way to use networkType and generationHash is wrong. It is in a fixed value